src/query/api/v1/handler/placement/get_test.go

```diff
@@ -113,4 +116,26 @@ func TestPlacementGetHandler(t *testing.T) {
 
 	resp = w.Result()
 	assert.Equal(t, http.StatusNotFound, resp.StatusCode)
+
+	// With bad version request
+	w = httptest.NewRecorder()
+	req = httptest.NewRequest("GET", "/placement/get?version=foo", nil)
```

Comment:
It doesn't affect the test outcome, but as it's configured now, shouldn't the request actually be a `GET` on `/placement` directly?